### ISSUE TITLE
Repeated "TypeError: document.querySelector(...) is null" in console

### ISSUE THREAD

**Issue by etekweb** [lightbeam24/customtube#2]
This seems to cause added lag on the YouTube site.

Console logs from youtube.com:

```
[main.js:1532:14](moz-extension://0edcb5d9-321a-471f-8913-edb711bb2142/main.js)
TypeError: document.querySelector(...) is null
    watchPageEveryLoad moz-extension://0edcb5d9-321a-471f-8913-edb711bb2142/main.js:1532
    wait moz-extension://0edcb5d9-321a-471f-8913-edb711bb2142/main.js:38
    (Async: setInterval handler)
    waitFor moz-extension://0edcb5d9-321a-471f-8913-edb711bb2142/main.js:33
    urlListen moz-extension://0edcb5d9-321a-471f-8913-edb711bb2142/main.js:818
```

Seems to be from this line in main.js:
```
if (notNeedNewString == true) {
		  document.querySelector("#bt-desc-upload-date #published-on").style.display = "none";	
```

Browser: Firefox 112.0.2
Layout Select: Disabled

**Comment by lightbeam24** [comment]
I think I found the fix. It'll be in update 1.0.2.

**Comment by lightbeam24** [comment]
Fixed as of 1.0.2.
